common/array.h

```diff
@@ -82,13 +82,16 @@ class Array {
 	}
 
 	/**
-	 * Construct an array with `count` copies of elements with value `value`.
+	 * Construct an array with @p count copies of elements with value @p value.
 	 */
 	Array(size_type count, const T &value) : _size(count) {
 		allocCapacity(count);
 		uninitialized_fill_n(_storage, count, value);
 	}
 
+	/**
+	 * @todo Doc required
+	 */
```

Comment:
Construct an array as a copy of the given @p array.

I think this should be explicit enough for developers reading the doc. But if we wanted more details we could say:
Construct an array with the same size as the given @p array and whose elements are copies of the elements in @p array.